examples/all-clusters-app/ameba/main/include/static-supported-modes-manager.h

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
All those static-supported-modes-manager.h/cpp seem copied and pasted for each platform. Why not have one static-supported-modes-manager.h/cpp in the common folder

Reply:
I'm not sure why, but there are already shared code that are not put in the all-clusters-common folder, For example, the definition of `bool lowPowerClusterSleep()` can be found in every platform, even though they are the same. I guess the reason is to allow different platform to provide different definitions, based on their different limitations.

Reply:
I guess we could add a layer for specific platform implementation. I will approve it but let see what other reviewers think of it